keyboards/jj40/keymaps/ajp10304/readme.md

```diff
@@ -1,11 +1,15 @@
 # AJP10304 Custom JJ40 Layout
-# Also available for the Atreus50 and Planck
+# Also available for the Planck, Shark and Atreus50
 
 **Note:** In the tables below where there are two characters on a key,
 the second is the output when shift is applied.
 
 **Note:** The below tables assume a UK layout.
 
+#### Flashing
+Use sleep to get a chance to get into boot mode.
+`sleep 5; bootloadHID -r .build/jj40_ajp10304.hex`
```

Comment:
`qmk flash -kb <keyboard> -km <keymap>` or `make keyboard:keymap:flash` is the preferred way to flash.

Reply:
I can't test this at the moment, because I don't have access to my JJ40 right now.  I remember never being able to get anything else to work, but that is going back a couple of years.

Reply:
The `:flash` target pulls the bootloader info from the keyboard (or keymap) config, and chooses the correct target. 

Additionally, if you tried a year or more ago... then the bootloadHID support may not have been added yet.  It is there now, and it has been tested.